test/clt-tests/native-support-for-message-queues/test-native-support-for-message-queues.rec

```diff
@@ -0,0 +1,628 @@
+|   45 | The Best Things In Life Are Free              | ISO 24680:1357  | %{DATETIME} | 20,25,30 |      1690761600 |
+|   46 | Money Doesn't Grow On "Trees                  | ISO 97531:86420 | %{DATETIME} | 20,25,30 |      1690761600 |
+|   47 | No Pain, No Gain                              | ISO 13579:86420 | %{DATETIME} | 20,25,30 |      1690761600 |
+|   48 | When In Rome, Do As The Romans Do             | ISO 56789:12345 | %{DATETIME} | 20,25,30 |      1690761600 |
+|   49 | Better Late Than Never                        | ISO 86420:97531 | %{DATETIME} | 20,25,30 |      1690761600 |
+|   50 | All's Well That Ends Well                     | ISO 13579:86420 | %{DATETIME} | 20,25,30 |      1690761600 |
++------+-----------------------------------------------+-----------------+---------------------+----------+-----------------+
+––– input –––
+docker exec kafka kafka-consumer-groups.sh --bootstrap-server localhost:9092 --group manticore_combined --reset-offsets --to-latest --topic my-data --execute > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker exec manticore mysql -h0 -P9306 -e "CREATE SOURCE kafka_combined (id bigint, term text, abbrev text, GlossDef json, location json, metadata json, tags json, timestamp_unix timestamp) type='kafka' broker_list='kafka:9092' topic_list='my-data' consumer_group='manticore_combined' num_consumers='1' batch=50;"; echo $?
+––– output –––
+0
+––– input –––
+docker exec manticore mysql -h0 -P9306 -e "CREATE TABLE destination_kafka_combined (id bigint, name text, short_name text, received_at text, size multi, lat float, lon float, views int, info text, tags json, timestamp_combined timestamp);"; echo $?
+––– output –––
+0
+––– input –––
+docker exec manticore mysql -h0 -P9306 -e "CREATE MATERIALIZED VIEW view_table_combined TO destination_kafka_combined AS SELECT id, term AS name, abbrev AS short_name, UTC_TIMESTAMP() AS received_at, GlossDef.size AS size, location.lat AS lat, location.lon AS lon, metadata.views AS views, metadata.info AS info, tags, timestamp_unix AS timestamp_combined FROM kafka_combined WHERE views > 1000 AND timestamp_combined >= 1690761600 AND tags IN ('item1', 'item2') AND lat > 50 AND lon > 5;"; echo $?
+––– output –––
+0
+––– input –––
+docker exec kafka ./import.sh; echo $?
+––– output –––
+0
+––– input –––
+timeout 60 bash -c 'docker exec manticore bash -c "tail -f /var/log/manticore/searchd.log" | grep -m2 "REPLACE+INTO+destination_kafka_combined" > /dev/null && echo "Download completed."' || echo "Download failed."
+––– output –––
+Download completed.
+––– input –––
+docker exec manticore mysql -h0 -P9306 -e "SELECT COUNT(*) FROM destination_kafka_combined;"
+––– output –––
++----------+
+| count(*) |
++----------+
+|        5 |
++----------+
+––– input –––
+docker exec manticore mysql -h0 -P9306 -e "SELECT * FROM destination_kafka_combined WHERE views > 1000 AND timestamp_combined >= 1690761600 AND tags IN ('item1', 'item2') AND lat > 50 AND lon > 5 ORDER BY id ASC;"
```

Comment:
Why do we have filters here? We need to be sure that the desired amount of rows appears in the destination table and they have the expected structure. This condition hasn't anything related to queue testing

Reply:
Done